uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeaveRequestTest.php

```diff
@@ -712,15 +717,16 @@ public function testLeaveRequestShouldNotBeCreatedWhenAbsenceTypeIsNotActive() {
     ]);
   }
 
-  /**
-   * @expectedException CRM_HRLeaveAndAbsences_Exception_InvalidLeaveRequestException
-   * @expectedExceptionMessage Leave Request days cannot be greater than maximum consecutive days for absence type
-   */
   public function testNumberOfDaysOfLeaveRequestShouldNotBeGreaterMaxConsecutiveLeaveDaysForAbsenceType() {
     $absenceType = AbsenceTypeFabricator::fabricate([
       'max_consecutive_leave_days' => 2
     ]);
 
+    $this->setExpectedException(
+      'CRM_HRLeaveAndAbsences_Exception_InvalidLeaveRequestException',
```

Comment:
I like using the `class` keyword instead of strings for class names, like `CRM_HRLeaveAndAbsences_Exception_InvalidLeaveRequestException::class`